source/switching_to_live/before_you_switch_to_live/index.html.md.erb

```diff
@@ -44,24 +50,43 @@ to GOV.UK Pay.
 
 ## Request a live GOV.UK Pay account
 
-1. Sign in to your [GOV.UK Pay
-   account](https://selfservice.payments.service.gov.uk/login).
+Requesting a live account is the same regardless of whether you are a Stripe or non-Stripe user.
 
-1. Select __My services__.
+1. Go to the [Request a live account](https://pay-admin.cloudapps.digital/request-to-go-live) page on GOV.UK Pay and select __Start now__.
+
+1. Enter the name of your organisation and select __Continue__.
+
+1. Choose how you will process payments and select __Continue__.
+
+1. Select the __Accept legal terms__ checkbox and select __Continue__.
```

Comment:
is it clear enough that accepting our legal terms satisfies the requirement (mentioned above) to have a signed contract/ MOU with Pay before taking live payments?

Reply:
will clarify